DESCRIPTION

```diff
@@ -2,7 +2,7 @@ Package: annoFuse
 Type: Package
 Title: Annotation And Filtering Analysis For Fusion Calls
 Version: 0.2
```

Comment:
```suggestion
Title: annoFuse: an R Package to annotate, prioritize, and interactively explore putative oncogenic RNA fusions
Version: 1.0.0
```
- Should we add the proper title here? 
- This will be a major update to the version

Reply:
Makes total sense to me!

Reply:
As in Slack - I'd leave this for now to 0.3.0 - we can eventually bump it also later on!
Title change is fine for me 😉